core/modules/user/user.theme.inc

```diff
@@ -386,3 +386,63 @@ function template_preprocess_user_profile(&$variables) {
   $variables['theme_hook_suggestions'][] = 'user_profile__' . $account->uid;
   $variables['theme_hook_suggestions'][] = 'user_profile__' . $account->uid . '__' . $view_mode;
 }
+
+/**
+ * Process variables for user-simplified-page.tpl.php.
+ *
+ * The $variables array contains the following arguments:
```

Comment:
The syntax should rather start with:
```
@param array $variables
```
And then the comment.

Reply:
Thanks. Fixed.